servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -488,14 +494,18 @@ void RenderForwardMobile::_render_scene(RenderDataRD *p_render_data, const Color
 	if (draw_sky || draw_sky_fog_only) {
 		RENDER_TIMESTAMP("Render Sky");
 
-		CameraMatrix projection = p_render_data->cam_projection;
+		// TODO change this to supporting more then 2 view projections
```

Comment:
You already made most code N-views already, so I would go the extra mile and convert this one too.